components/console-backend-service/internal/gqlschema/schema.graphql

```diff
@@ -1014,6 +1014,82 @@ type VersionInfo {
     kymaVersion: String!
 }
 
+# Trigger
+
+type Trigger {
+    name: String!
+    namespace: String!
+    broker: String!
+    filterAttributes: JSON!
+    subscriber: Subscriber!
+    status: TriggerStatus!
+}
+
+type TriggerStatus {
+    reason: [String!]
+    status: TriggerStatusType!
+}
+
+enum TriggerStatusType {
+    FAILED
+    READY
+}
+
+type Subscriber {
+    uri: String
+    ref: SubscriberRef
+}
+
+type SubscriberRef {
+    apiVersion: String!
+    kind: String!
+    name: String!
+    namespace: String!
+}
+
+input SubscriberInput {
+    uri: String
+    ref: SubscriberRefInput
+}
+
+input SubscriberRefInput {
+    apiVersion: String!
+    kind: String!
+    name: String!
+    namespace: String!
```

Comment:
As I know, Trigger is namespaced resource, so namespace is necessary? I think no, so please remove it.

Reply:
OK, my fault. Namespace is necessary :D